Formula/automake.rb

```diff
@@ -15,20 +16,10 @@ class Automake < Formula
 
   depends_on "autoconf"
 
-  # Download more up-to-date config scripts.
-  resource "config" do
-    url "https://git.savannah.gnu.org/cgit/config.git/snapshot/config-0b5188819ba6091770064adf26360b204113317e.tar.gz"
-    sha256 "3dfb73df7d073129350b6896d62cabb6a70f479d3951f00144b408ba087bdbe8"
-    version "2020-08-17"
-  end
```

Comment:
Can you explain why this is no longer needed?

Reply:
I don't know, to be honest. I just inherited that from https://github.com/Homebrew/homebrew-core/pull/66511/commits/82f7491cdf202c2e8c7bab26071c7d6f64f9b8fb

Reply:
You can check what `config.guess`/`config.sub` version is installed. If it's older than 2020-08-17 then this is still needed.

Reply:
From `config.guess`:

```sh
#! /bin/sh
# Attempt to guess a canonical system name.
#   Copyright 1992-2020 Free Software Foundation, Inc.

timestamp='2020-11-07'
```

Same in `config.sub`. Is that adequate?

Reply:
Ah yes, it was included in automake 1.16.3 but the resource wasn't properly removed at the time. The autoconf relation confused me there - it's actually completely unrelated.